### PR TITLE
Test for README Example

### LINKED ISSUE #10: Write tests for examples in README to ensure we're not lying
Looks like #6 needs to be fixed before this one. **Edit: fixed!**

cc: @bjdelro

### PR DESCRIPTION
**Pull request by bjdelro**
Fixes #10

### CODE REVIEW

File: src/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"use strict";
```
Comment (jednano) [inline src/setupTests.js]: This file shouldn't have been generated here.

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,4 @@
 npm-debug.log*
 dist/
 node_modules/
+.idea*
```
Comment (jednano) [inline .gitignore]: This should be in your global git ignore.

**Review by jednano — CHANGES_REQUESTED** [review]
Nice work! Just a couple requests. Thanks!

File: src/createBEMBlock.test.tsx [hunk 3]
```diff
@@ -108,4 +109,41 @@ describe('createBEMBlock', () => {
 			<FooBlock block="foo" modifiers="mod2" />,
 		)).toMatchSnapshot()
 	})
+
+	it('Renders README example with correct className as shown', () => {
+		class Bar extends React.Component {
```
Comment (jednano) [inline src/createBEMBlock.test.tsx:114]: Whoops! Need to update the README to include `extends React.Component`.

File: src/createBEMBlock.test.tsx [hunk 4]
```diff
@@ -108,4 +109,41 @@ describe('createBEMBlock', () => {
...
+							garpley
+						</div>
+					</div>
+				)
+			}
+		}
+
+		const BarElement = createBEMElement(Bar);
+
+		class Foo extends React.Component {
```
Comment (jednano) [inline src/createBEMBlock.test.tsx:131]: same

File: src/createBEMBlock.test.tsx [hunk 5]
```diff
@@ -108,4 +109,41 @@ describe('createBEMBlock', () => {
...
+			static defaultProps = {
+				block: 'foo'
+			}
+			render() {
+				return (
+					<div modifiers="mod1 mod2">
+						<BarElement />
```
Comment (jednano) [inline src/createBEMBlock.test.tsx:138]: Whoops! Readme needs `<BarElement />` too.

File: src/createBEMBlock.test.tsx (hunk 3 above)
Comment (bjdelro) [reply src/createBEMBlock.test.tsx:114]: i can include the README updates

File: src/setupTests.ts [hunk 6]
```diff
@@ -1,4 +0,0 @@
-import { configure } from 'enzyme';
```
Comment (jednano) [inline src/setupTests.ts]: This file we want to keep. Keep all the TS files and ditch the one JS file.

**Review by jednano — APPROVED** [review]

**Pull request merged**